lib/Notes/NotesAccordion/NotesAccordion.js

```diff
@@ -1,102 +1,71 @@
 
-import NotesList from './components';
-import styles from './NotesAccordion.css';
-
-const defaultColumnsConfig = [
-  {
-    name: 'date',
-    title: <FormattedMessage id="stripes-smart-components.date" />,
-    width: '30%',
-  },
-  {
-    name: 'updatedBy',
-    title: <FormattedMessage id="stripes-smart-components.updatedBy" />,
-    width: '30%',
-  },
-  {
-    name: 'title',
-    title: <FormattedMessage id="stripes-smart-components.title" />,
-    width: '40%',
-  },
-];
+import NotesList from './components/NotesList';
+import NotesAssigningModal from './components/NotesAssigningModal';
 
-export const columnShape = PropTypes.shape({
-  name: PropTypes.string.isRequired,
-  title: PropTypes.node.isRequired,
-  width: PropTypes.string,
-});
-
-export const noteShape = PropTypes.shape({
-  id: PropTypes.string,
-  lastSavedDate: PropTypes.instanceOf(Date),
-  lastSavedUserFullName: PropTypes.string,
-  title: PropTypes.string,
-});
+import styles from './NotesAccordion.css';
 
-export class NotesAccordion extends Component {
+export default class NotesAccordion extends Component {
   static propTypes = {
-    columnsConfig: PropTypes.arrayOf(columnShape),
-    notes: PropTypes.arrayOf(noteShape),
     onCreate: PropTypes.func.isRequired,
-    onNoteClick: PropTypes.func.isRequired,
+    onSaveAssigningResults: PropTypes.func.isRequired,
     open: PropTypes.bool,
-  }
+  };
+
+  state = { modalIsOpen: false };
 
-  static defaultProps = {
-    columnsConfig: defaultColumnsConfig,
-    notes: [],
+  onCloseModalHandler = () => {
+    this.setState(() => {
+      return {
+        modalIsOpen: false,
+      };
+    });
+  };
+
+  onSaveAssigningResultsHandler = (changedNoteIdToStatusMap) => {
+    this.props.onSaveAssigningResults(changedNoteIdToStatusMap);
+    this.onCloseModalHandler();
   }
 
   renderHeader = () => {
     return (
-      <Headline
-        data-test-notes-accordion-header
-        size="large"
-        tag="h3"
-      >
+      <Headline size="large" tag="h3">
         <FormattedMessage id="stripes-smart-components.notes" />
       </Headline>
     );
-  }
+  };
 
   renderHeaderButtons() {
-    const {
-      onCreate,
-    } = this.props;
+    const { onCreate } = this.props;
 
     return (
       <Fragment>
-        <Button data-test-notes-accordion-add-button>
-          <FormattedMessage id="stripes-smart-components.add" />
+        <Button onClick={this.onAssignButtonClickHandler}>
+          <FormattedMessage id="stripes-smart-components.assign" />
         </Button>
-        <Button
-          data-test-notes-accordion-new-button
-          buttonClass={styles['new-button']}
-          onClick={onCreate}
-        >
+        <Button buttonClass={styles['new-button']} onClick={onCreate}>
           <FormattedMessage id="stripes-smart-components.new" />
         </Button>
       </Fragment>
     );
   }
 
+  onAssignButtonClickHandler = () => {
```

Comment:
Is "handler" postfix needed?. Also, maybe this wouldn't be needed if you had `toggleModal` method.